doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+        and black relative exposures.
+      </para>
+    </sect5>
+
+    <sect5 id="filmicrgb-contrast">
+      <title>Contrast (<guilabel>Look</guilabel> tab)</title>
+      <para>
+        The filmic rgb S-curve is created from the user parameters, by computing the position of
+        virtual nodes and interpolating them, similarly to the tone curve module (but here, the
+        nodes cannot be moved manually). The filmic rgb S-curve is split into three parts: a
+        middle linear part, and two extremities that transition smoothly from the slope of the
+        middle part to the ends of the exposure range.
+      </para>
+      <para>
+        The
+        <guilabel>contrast</guilabel>
+        slider controls the slope of the middle part of the curve, as illustrated in the graph
+        display.
+      </para>
+      <para>
+        The contrast parameter drives the slope of the central part of the curve. The larger the
+        dynamic range is, the greater the contrast should be set. This parameter mostly affects
+        mid-tones.
+      </para>
+      <para>
+        When the contrast is set to 1, this disables the S-curve.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>Latitude (<guilabel>Look</guilabel> tab)</title>
+      <para>
+        The latitude is the range between the 2 nodes enclosing the central linear portion of
+        the curve, expressed as a percentage of the dynamic range defined in the
+        <guilabel>scene</guilabel>
+        tab (white-relative exposure minus black-relative-exposure). It is the luminance range
+        that is remapped in priority, and it is remapped to the luminance interval defined by
+        the <link linkend="filmicrgb-contrast"
+          >Contrast</link> parameter. It is
+        usually advised to keep the latitude as large as possible, while avoiding clipping. If
+        clipping is observed, you can compensate the effects by either decreasing the latitude,
+        shifting the latitude interval with the
+        <guilabel>shadow/highlights balance</guilabel>
+        parameter, or by decreasing the contrast.
+      </para>
+      <para>
+        The latitude also defines the range of luminances that is not desaturated at the
+        extremities of the luminance range (see <xref linkend="filmicrgb-saturation"/>).
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>Shadows/highlight balance (<guilabel>Look</guilabel> tab)</title>
+      <para>
+        By default, the latitude is centered in the middle of the dynamic range. If this
+        produces clipping in one part of the other of the curve, the balance parameter allows to
+        slide the latitude along the slopes, towards the shadows or towards the highlights. This
+        allows to give more room to one extremity of the dynamic range than to the other, if the
+        image properties demand it.
+      </para>
```

Comment:
This allows more room to be given to one extremity of the dynamic range than to the other, if the image properties demand it.